packages/medusa/src/services/new-totals.ts

```diff
@@ -172,8 +172,7 @@ export default class NewTotalsService extends TransactionBaseService {
       subtotal = 0 // in that case we need to know the tax rate to compute it later
```

Comment:
comment(this-file): I believe we are using the `unit_amount` several places in the new totals service

Reply:
let me check

Reply:
done, thanks 🙏

Reply:
Did you run a file search? I can still find two occurrences :)

Reply:
I looked for all unit_amount, let me double check

Reply:
<img width="1291" alt="Screenshot 2023-02-13 at 11 45 32" src="https://user-images.githubusercontent.com/25098370/218437414-29785010-a318-4c3d-a6da-fd80e33f0f16.png">

Reply:
I ll look every where